cumulusci/tests/util.py

```diff
@@ -0,0 +1,61 @@
+import os
+import random
+
+import yaml
+
+from cumulusci.core.config import BaseGlobalConfig
+from cumulusci.core.config import BaseProjectConfig
+from cumulusci.core.keychain import BaseProjectKeychain
+from cumulusci.core.config import OrgConfig
+from cumulusci import __location__
+
+def random_sha():
```

Comment:
I'm not quite sure the implications of doing that but I'm open to discussing it.  The functionality works fine without a consistent random sequence each test run.